### ISSUE TITLE
Box Node Code

### ISSUE THREAD

**Issue by nhadlaw** [touretzkyds/KnowledgeGraphDemo#61]
I'm creating this issue because along with fixing the box node colors, I wanted to clean up the box node code to make it more generalized. This shouldn't take too long but I think it will help us down the road when we have a lot more bnodes in our data.

**Comment by nhadlaw** [comment]
@touretzkyds  Do we still want to keep the box node names starting with 'b' and then a number? Or should we create something more general? For example, when a user sees b3: [avgmass, avglength, avgLIfespan], they then see b4:[avgmass, avgLength], then they see b5:[avgLength]. A user won't really understand what 'b3' or 'b4' really mean even if we signify that it's a box node it's still unclear what that really means. What if instead, we labeled the unexpanded b3 node more appropriately to be something like:  'Alligator Mississippiensis Comparisons: [avgmass, avglength, avgLIfespan]' and then the unexpanded b4 node would be: 'Alligator Mississippiensis Comparisons: [avglength, avgLIfespan]' and so forth?

**Comment by touretzkyds** [comment]
I don't think lengthy labels like that will fit in the node. And I don't think they're necessary. The link labels already say prefComparisons, and they can seethat this property is hanging off the Alligator node. Let's stick with the b3/b4/b5 labels for now.

**Comment by nhadlaw** [comment]
Ok sounds good. I wrote a lot more generalized code for displaying the box nodes. I did this so in the future when we have more and more of them, it would be easier to add them to the graph without breaking any code. I didn't worry too much about the layout but I created a function called 'add_new_bnodes' that takes care of continously adding bnodes from the box node query until there is non left. This way it will work regardless of if we have 3 or 20 box nodes.
